faqs/galaxy/tools_tool_doesn't_recognize_any_input_datasets.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Tool doesn't recognize any input datasets
+area: tools
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+1. Convert Format or modify the datatype using pencil icon {% icon galaxy-pencil %} to reach the Edit Attributes form.
```

Comment:
These instructions aren't very clear (not your fault), and also don't seem to correspond to the title. Not sure what this FAQ was trying to.